addons/l10n_dk/data/account_tax_report_data.xml

```diff
@@ -0,0 +1,152 @@
+    </record>
+    <record id="account_tax_report_line_vat_statement_sales_tax" model="account.tax.report.line">
+        <field name="name">Salgsmoms (udgående moms)</field>
+        <field name="tag_name">Salgsmoms (udgående moms)</field>
+        <field name="sequence" eval="1"/>
+        <field name="parent_id" ref="l10n_dk.account_tax_report_line_vat_statement_sales_group"/>
+        <field name="country_id" ref="base.dk"/>
+    </record>
+    <record id="account_tax_report_line_vat_statement_international_purchase_products" model="account.tax.report.line">
+        <field name="name">Moms af varekøb i udlandet (både EU og lande uden for EU)</field>
+        <field name="tag_name">Moms af varekøb i udlandet (både EU og lande uden for EU)</field>
+        <field name="sequence" eval="2"/>
+        <field name="parent_id" ref="l10n_dk.account_tax_report_line_vat_statement_sales_group"/>
+        <field name="country_id" ref="base.dk"/>
+    </record>
+    <record id="account_tax_report_line_vat_statement_international_purchase_services" model="account.tax.report.line">
+        <field name="name">	Moms af ydelseskøb i udlandet med omvendt betalingspligt</field>
+        <field name="tag_name">	Moms af ydelseskøb i udlandet med omvendt betalingspligt</field>
+        <field name="sequence" eval="3"/>
+        <field name="parent_id" ref="l10n_dk.account_tax_report_line_vat_statement_sales_group"/>
+        <field name="country_id" ref="base.dk"/>
+    </record>
+
+    <record id="account_tax_report_line_vat_statement_deduction_group" model="account.tax.report.line">
+        <field name="name">Fradrag</field>
+        <field name="code">Momsangivelse</field>
+        <field name="sequence" eval="2"/>
+        <field name="parent_id" ref="l10n_dk.account_tax_report_line_vat_statement"/>
+        <field name="country_id" ref="base.dk"/>
+    </record>
+    <record id="account_tax_report_line_vat_statement_deduction_purchase_tax" model="account.tax.report.line">
+        <field name="name">Købsmoms (indgående moms)</field>
+        <field name="tag_name">Købsmoms (indgående moms)</field>
+        <field name="sequence" eval="1"/>
+        <field name="parent_id" ref="l10n_dk.account_tax_report_line_vat_statement_deduction_group"/>
+        <field name="country_id" ref="base.dk"/>
+    </record>
+    <record id="account_tax_report_line_vat_statement_deduction_oil_bottle_tax" model="account.tax.report.line">
+        <field name="name">Olie- og flaskegasafgift</field>
+        <field name="tag_name">Olie- og flaskegasafgift</field>
+        <field name="sequence" eval="2"/>
+        <field name="parent_id" ref="l10n_dk.account_tax_report_line_vat_statement_deduction_group"/>
+        <field name="country_id" ref="base.dk"/>
+    </record>
+    <record id="account_tax_report_line_vat_statement_deduction_electrical_tax" model="account.tax.report.line">
+        <field name="name">Elafgift</field>
+        <field name="tag_name">Elafgift</field>
```

Comment:
It is easier for configuration purposes to keep a shorter tag_name.  (I don't know if there is e.g. a number related to the box to be filled in in the declaration)

Reply:
I have shortened some of the names. There is not really a number that I know of, but if the new names are still too long, I can try to find out if there are maybe some old shortcodes or numbers that can be used instead